src/App.js

```diff
@@ -1,15 +1,21 @@
-import './App.css';
+import React from 'react';
+import { HashRouter, Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Header from './components/header';
+import Quote from './components/qoute';
+import Home from './components/home';
+import Footer from './components/footer';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Calculator App</h1>
-      </header>
-      <Calculator />
-    </div>
-  );
-}
+const App = () => (
+  <HashRouter>
+    <Header />
+    <Routes>
+      <Route exact path="/" element={<Home />} />
+      <Route path="/calculator" element={<Calculator />} />
```

Comment:
- [ ] As seen in the Wireframe. Please ensure that there is a text to the left of your calculator is shown in your project.


| First Header  | Second Header |
| ------------- | ------------- |
| Wireframe  | ![image](https://user-images.githubusercontent.com/52934128/212908922-4fc0a339-4f25-4b9d-964e-0dcedaee204f.png)  |
| Your Design  | <img width="960" alt="image" src="https://user-images.githubusercontent.com/52934128/212909952-5efe628b-3244-4470-ba57-2f6739bd7444.png"> |